src/core/charpak_mod.F90

```diff
@@ -1,16 +1,8 @@
-!------------------------------------------------------------------------------
-!                  CATChem Model                                              !
-!------------------------------------------------------------------------------
-!BOP
-!
-! !MODULE: charpak_mod.F90
-!
-! !DESCRIPTION: Module CHARPAK\_MOD contains routines from the CHARPAK
-!  string and character manipulation package used by GEOS-Chem.
-!\\
-!\\
-! !INTERFACE:
-!
+!> \file charpak_mod.F90
```

Comment:
```suggestion
!> \file
```
It will default to current file [it seems](https://www.doxygen.nl/manual/commands.html#cmdfile).